var/spack/repos/builtin/packages/libf77zmq/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+
+class Libf77zmq(MakefilePackage):
```

Comment:
It might be better to call this package `f77-zmq`. Thoughts?

Reply:
I am very new to Spack, so I am not aware of the naming conventions.
I agree with you that f77-zmq is nicer.
However, it seems to me that most of the libraries have names starting with "lib" (although this is not true for zlib, alsa-lib, and a few others), and the produced files here are `libf77zmq.so` and `libf77zmq.a`.
So I don't know what is best, and I leave the responsibility of the name of the package to you ;-)

Reply:
We usually go with whatever the developers call the package. For example, zlib is called zlib, not libz. For many of the X11 libraries, the developers call them `libx11`. Since this developer seems to call the package `f77-zmq`, I would recommend that name. It also prevents someone from adding a duplicate package, since `spack create` automatically detects that name.

Reply:
OK. I understand. I will change the name then.